### ISSUE TITLE
Field test REDCapTidieR with current complex databases

### ISSUE THREAD

**Issue by skadauke** [CHOP-CGTDataOps/REDCapTidieR#5]
# Feature Request Description
Before we publicize this package we should make sure that things look as expected when we use it on one or two actual complex REDCap databases. The two I am thinking of are the draft Prodigy huCART19 database and the BMT Outcomes database. 

# Proposed Solution
Let's make sure that the import of those two databases works as expected:

- [ ] No errors pop up when the database is imported
- [ ] Each instrument is represented as a tibble
- [ ] Each instrument has the expected partial keys
- [ ] Each instrument has the expected fields
- [ ] Partial keys are never `NA`
- [ ] There are no rows that are all `NA` in data fields (not sure if this is what we expect)?

Since most of these steps are really a QA activity, if you like, you can save the data as CSV and give to Amir to review and report back.

It may also be worth writing automated tests for some/most of these, just to nail down expected behavior. 

#### Check Against Sample REDCap Databases for `REDCapR`

`REDCapTidieR` should function as expected when tested against the same databases that `REDCapR` uses:

> https://github.com/OuhscBbmc/REDCapR/blob/main/inst/misc/example.credentials 

### Checklist

- [X] The issue is atomic
- [X] The issue description is documented
- [X] The issue title describes the problem succinctly
- [X] Developers are assigned to the issue
- [X] Labels are assigned to the issue

**Comment by skadauke** [comment]
Feel free to close this once Amir is done with his review.

**Comment by rsh52** [comment]
To remain open until #8 is resolved, and Amir checks label behavior.

**Comment by skadauke** [comment]
Let's also take a look at the CART Longitudinal DB... I bet we'll find another issue or two. @rsh52, could you send CSV files from that database to Amir once we've implemented the current round of PRs?

**Comment by skadauke** [comment]
> Let's also take a look at the CART Comprehensive DB... I bet we'll find another issue or two. @rsh52, could you send CSV files from that database to Amir once we've implemented the current round of PRs?

@rsh52 Can you send those CSVs to me (instead of Amir) so I can take a look?

**Comment by rsh52** [comment]
Sure thing, will send as soon as the current PRs are merged and the actions pass. 👍

**Comment by skadauke** [comment]
Looked through the CART DB, and all looks good!

**Comment by rsh52** [comment]
Excellent news! 🎉
